src/pages/Admin/Venue/DetailsForm.tsx

```diff
@@ -623,13 +623,18 @@ const DetailsFormLeft: React.FC<DetailsFormLeftProps> = (props) => {
           )}
         <div className="input-container">
```

Comment:
[OOS, future] We could do some nice stuff around parsing different things here, and ensuring we get the parentVenueId we want out of it, eg. extract it if they paste a full link/etc, check if that venue ID is valid, etc.